### PR TITLE
PUT for /quiz

### PR DESCRIPTION
**Pull request by Mark-Jung**
Can now edit quiz

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 @app.route('/')
 def hello_world():
-    return 'Goodbye ADC!'
+    return "Aeng?!"
```
Comment (jameswhang) [inline app.py]: lol

File: controllers/quiz.py [hunk 2]
```diff
@@ -26,6 +27,21 @@ def make_quiz(cls, quiz_name, questions):
                 return error_message, status, None
 
         return "", 201, None
+    
+    @classmethod
+    def update_quiz(cls, id, quiz_name, questions):
```
Comment (jameswhang) [inline controllers/quiz.py]: ```id``` is a keyword in Python. You can override it, but it's a bad convention to do so. Rename it to quiz_id

File: controllers/quiz.py [hunk 3]
```diff
@@ -45,6 +61,25 @@ def make_question(cls, question, quiz_id):
                 return error_message, status
         return "", 201
 
+    @classmethod
+    def update_question(cls, question):
+        req_params = ['id', 'question_text', 'cards']
```
Comment (jameswhang) [inline controllers/quiz.py]: Can you do this check on the view side?

File: tests/helper_tests.py [hunk 4]
```diff
@@ -71,11 +71,11 @@ def get_purchase_history(cls, inst, email):
                     '/purchase/' + email,
                 )
 
-    @classmethod
-    def get_balance(cls, inst, mode):
-        return inst.app.get(
-                    '/payment/' + mode,
-        )
+    # @classmethod
```
Comment (jameswhang) [inline tests/helper_tests.py]: Any reason you left this as a comment instead of blowing it up?

Comment (Mark-Jung) [reply tests/helper_tests.py]: I forgot about it after seeing that all tests passed lolol mb

File: controllers/quiz.py (hunk 3 above)
Comment (Mark-Jung) [reply controllers/quiz.py]: Sure

File: controllers/quiz.py (hunk 2 above)
Comment (Mark-Jung) [reply controllers/quiz.py]: Alright. I'll edit the following 'id' to 'quiz_id', 'question_id', and 'quizcard_id'.

File: app.py (hunk 1 above)
Comment (Mark-Jung) [reply app.py]: Aeng

**Comment by myidsan** [comment]
LGTM after editing the 'id's to 'quiz_id', 'question_id', and 'quizcard_id'

**Comment by Mark-Jung** [comment]
@jameswhang PTAL. League today?

**Comment by jameswhang** [comment]
@Mark-Jung Fix conflict

**Pull request merged**